packages/web/src/components/track/PremiumTrackSection.tsx

```diff
@@ -112,6 +122,11 @@ const LockedPremiumTrackSection = ({
     ? FollowSource.HOW_TO_UNLOCK_MODAL
     : FollowSource.HOW_TO_UNLOCK_TRACK_PAGE
   const account = useSelector(getAccountUser)
+  const isUSDCGated = isPremiumContentUSDCPurchaseGated(premiumConditions)
```

Comment:
nit: Feels like we should stick to either `isPurchaseGated` or `isUSDCGated`, seeing a mix of both in this pr

Reply:
Updated all instances to be `isUSDCPurchaseGated`